src/styles/common/_mixins.scss

```diff
@@ -119,8 +119,7 @@
 @mixin input-style {
     box-shadow: none;
     min-height: $input-height !important;
-    height: $input-height;
```

Comment:
I think we decided to keep height as is rt ?

Reply:
yes we already have min-height set ..we dont need height separately..it will be minimum 40px